mitiq/pt/pt.py

```diff
@@ -12,40 +12,44 @@
 from mitiq import QPROGRAM, Executor, Observable, QuantumResult
 from mitiq.interface import atomic_converter
 
-# Paulis = Union[cirq.IdentityGate, cirq.XPowGate, cirq.YPowGate, cirq.ZPowGate]
-
-
-def _generate_lookup_table(
-    gate: str,
-) -> List[Tuple[cirq.Gate, cirq.Gate, cirq.Gate, cirq.Gate]]:
-    if gate not in ["CNOT", "CZ"]:
-        raise ValueError("Invalid two-qubit gate. Supported gates: CNOT, CZ")
-
-    c = cirq.X if gate == "CNOT" else cirq.Z
-    lookup_table = [
-        (cirq.I, cirq.I, cirq.I, cirq.I),
-        (cirq.X, cirq.I, cirq.I, cirq.X),
-        (cirq.Y, cirq.I, cirq.I, cirq.Y),
-        (cirq.Z, cirq.I, cirq.I, cirq.Z),
-        (cirq.I, c, c, cirq.I),
-        (cirq.X, c, c, cirq.X),
-        (cirq.Y, c, c, cirq.Y),
-        (cirq.Z, c, c, cirq.Z),
-        (cirq.I, cirq.Y, cirq.Y, cirq.I),
-        (cirq.X, cirq.Y, cirq.Y, cirq.X),
-        (cirq.Y, cirq.Y, cirq.Y, cirq.Y),
-        (cirq.Z, cirq.Y, cirq.Y, cirq.Z),
-        (cirq.I, cirq.Z, cirq.Z, cirq.I),
-        (cirq.X, cirq.Z, cirq.Z, cirq.X),
-        (cirq.Y, cirq.Z, cirq.Z, cirq.Y),
-        (cirq.Z, cirq.Z, cirq.Z, cirq.Z),
-    ]
-
-    return lookup_table
-
-
-lookup_table_CNOT = _generate_lookup_table("CNOT")
-lookup_table_CZ = _generate_lookup_table("CZ")
+# P, Q, R, S from https://arxiv.org/pdf/2301.02690.pdf
+CNOT_twirling_gates = [
+    (cirq.I, cirq.I, cirq.I, cirq.I),
+    (cirq.I, cirq.X, cirq.I, cirq.X),
+    (cirq.I, cirq.Y, cirq.Z, cirq.Y),
+    (cirq.I, cirq.Z, cirq.Z, cirq.Z),
+    (cirq.Y, cirq.I, cirq.Y, cirq.X),
+    (cirq.Y, cirq.X, cirq.Y, cirq.I),
+    (cirq.Y, cirq.Y, cirq.X, cirq.Z),
+    (cirq.Y, cirq.Z, cirq.X, cirq.Y),
+    (cirq.X, cirq.I, cirq.X, cirq.X),
+    (cirq.X, cirq.X, cirq.X, cirq.I),
+    (cirq.X, cirq.Y, cirq.Y, cirq.Z),
+    (cirq.X, cirq.Z, cirq.Y, cirq.Y),
+    (cirq.Z, cirq.I, cirq.Z, cirq.I),
+    (cirq.Z, cirq.X, cirq.Z, cirq.X),
+    (cirq.Z, cirq.Y, cirq.I, cirq.Y),
+    (cirq.Z, cirq.Z, cirq.I, cirq.Z),
+]
+# TODO: check the CZ gates. should be in PQRS order
```

Comment:
I noticed when modifying the lookup tables that the gates were not in $P, Q, R, S$ order. I think that makes most sense since that's what the reference we are referring to uses. Could you ensure the `CZ_twirlings_gates` object is defined in that order as well, and then fill in the `test_twirl_CZ` test as well?